### PR TITLE
Towards Python 3 Support

### PR DESCRIPTION
**Pull request by DLu**
New version of #10 based off new branch

### CODE REVIEW

**Review by JafarAbdi — APPROVED** [review]

File: body/stretch_body/stepper.py [hunk 1]
```diff
@@ -564,7 +564,7 @@ def menu_transaction(self,x,do_print=True):
             while self.transport.ser.inWaiting():
                 r=self.transport.ser.readline()
                 if do_print:
-                    print r,
+                    print(r)
```
Comment (mamoll) [inline body/stretch_body/stepper.py:567]: This prints a new line, while the original did not. You can fix that by adding this import and changing the print command to:
```python
from __future__ import print_function
...
print(r, end=' ')
```

**Review by mamoll — APPROVED** [review]
Looks good to me, modulo one small nit. Have you tried running `2to3` to see if other changes are suggested?

**Comment by hello-binit** [comment]
Merged here: https://github.com/hello-robot/stretch_body/tree/feature/towards_python3

**Comment by DLu** [comment]
I'm confused why you closed this. The branch you linked to isn't merged into anything and is the same as this branch.

**Comment by hello-binit** [comment]
Were you still working on this PR? I plan to merge commits into the branch linked above.

**Comment by DLu** [comment]
Okay, just be sure to include the commit I made yesterday: https://github.com/PickNikRobotics/stretch_body/commit/5dbcdef70647f15a97467a8611f79fd6a49d2410

**Pull request closed without merging**